napari/resources/_icons.py

```diff
@@ -65,3 +66,103 @@ def get_colorized_svg(
     # use regex to find the svg tag and insert css right after
     # (the '\\1' syntax includes the matched tag in the output)
     return svg_elem.sub(f'\\1{svg_style.format(color, opacity)}', xml)
+
+
+def generate_colorized_svgs(
+    svg_paths: Iterable[Union[str, Path]],
+    colors: Iterable[Union[str, Tuple[str, str]]],
+    opacities: Iterable[float] = (1.0,),
+    theme_override: Optional[Dict[str, str]] = None,
+) -> Iterator[Tuple[str, str]]:
+    """Helper function to generate colorized SVGs.
+
+    This is a generator that yields tuples of ``(alias, icon_xml)`` for every
+    combination (cartesian product) of `svg_path`, `color`, and `opacity`
```

Comment:
```suggestion
    combination (Cartesian product) of `svg_path`, `color`, and `opacity`
```

More respect for M. Descartes please.